test/functional/filters_controller_test.rb

```diff
@@ -68,4 +68,23 @@ class FiltersControllerTest < ActionController::TestCase
 
     assert_select "table[data-table='inline']"
   end
+
+  test 'should disable overriding and start inheriting taxonomies from roles' do
+    permission1 = FactoryGirl.create(:permission, :domain, :name => 'permission1')
+    role = FactoryGirl.build(:role, :permissions => [])
+    role.add_permissions! [permission1.name]
+    org1 = FactoryGirl.create(:organization)
+    org2 = FactoryGirl.create(:organization)
+    role.organizations = [ org1 ]
+    role.filters.reload
+    filter_with_org = role.filters.detect { |f| f.allows_organization_filtering? }
```

Comment:
It would only matter really, if this type of code would be majority of things we do. That's why I don't like speed arguments until we are making something faster: this would not be noticeable anywhere. I have nothing against `&:allows_organization_filtering?`, I just don't agree this kind of reasons should be something to base our decision if we enable some cop or not.